package.json

```diff
@@ -14,9 +14,11 @@
     },
     "homepage": "https://github.com/prezly/uploadcare#readme",
     "scripts": {
-        "build": "tsc --project .",
+        "build:main": "tsc --project .",
+        "build:umd": "webpack --output-filename index.js",
```

Comment:
Should we also add `browser` property to package.json? 

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#browser

Reply:
As far as I understood, it should only be used instead of `main` property, and not in parallel.

Reply:
Yeah, you're right. Thanks!